.azuredevops/modulePipelines/ms.aad.domainservices.yml

```diff
@@ -20,7 +20,7 @@ trigger:
   paths:
```

Comment:
As it was just merged, let's also include the new `Microsoft.DataProtection.Backupvaults` module

Reply:
Done.